static/public_pages/style.css

```diff
@@ -120,16 +120,15 @@ button {
   line-height: 1.75;
   letter-spacing: 0.02857em;
   border: 0;
-  padding: 6px 16px;
+  padding: 4px 10px;
   border-radius: 4px;
   color: rgba(0, 0, 0, 0.87);
   background-color: #e0e0e0;
   font-family: "Roboto", "Helvetica", "Arial", sans-serif;
-  transition:
-    background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,
-    box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,
-    border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,
-    color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
+  transition: background-color 0.3s;
```

Comment:
someone gave up on animations hahaha

Reply:
Ah ah yes didn't see the difference between the two.